kymatio/utils.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+import torch
+
+
+class ScatteringTransformer(BaseEstimator, TransformerMixin):
+    def __init__(self, S, signal_shape, frontend='numpy'):
+        """Creates an object that is compatible with the scikit-learn API
+        and implements the .transform method.
+        
+        Parameters
+        ==========
+
+        S: an instance of Scattering1D, Scattering2D or Scattering3D
```

Comment:
Why can't this be fixed by the class itself? In other words, we'd have `Scattering1D`, `Scattering2D`, etc., just like we have for Keras?

Reply:
I agree!